### DISCUSSION TITLE
Total freshwater vs anomaly

### DISCUSSION THREAD

**Discussion by mankoff** [NASA-GISS/freshwater-forcing-workshop#18] [Timeseries development]
Per earlier email discussion, and based on what was used in the Schmidt /et al./ 2023 paper (doi:10.1029/2023gl106530) we should be generating an *anomaly* product. This complicates things. It forces a variety of decisions to be made about the physical form of the expected anomaly, location of anomaly, and necessitates discarding information - and in general I try to avoid information loss. It also frames other discussions we're having about icebergs vs. liquid runoff.

First and by definition of the word 'anomaly' we need a baseline. I think the baseline is 'mass loss = 0 for a reference period'. If the baseline is 'mass' but the anomaly is, per the workshop title, 'freshwater', how do we relate these? These are two different things and cannot always be easily compared.

How do we define 'anomaly'? Is it just the negative mass term relative to the baseline?

Some examples

+ There are locations with 0 mass change but large freshwater discharge (offset by equally large mass gain from snowfall). Or more extreme, large runoff but net mass gain. If both snowfall and discharge increase, there is a freshwater anomaly in reality, but not easily detectable. Or if discharge remains steady but snowfall decreases, there is no freshwater anomaly, but it appears that there is one.

  
+ What form does the anomaly take? Liquid runoff or solid calving/iceberg? Models (generally) have runoff but not calving, suggesting the anomaly should be icebergs. But surface melt is the dominant change in the observational record. And much of our discussions elsewhere are on liquid inputs.

 
+ SW Greenland has a large mass loss signal (baseline) and change (anomaly), but is almost entirely land terminating, meaning no icebergs. Where should we put the icebergs spatially if not there? Perhaps where discharge data shows the most icebergs? But that's not where the mass anomaly is, so we've spatially decoupled the anomaly from the baseline.

  
+ @jlbamber has a 2018 dataset, currently being updated, that is 'freshwater into the arctic'.
  + Do we discard all non-Greenlandic sources?
  + Keep all ice (Greenland, Canada, Iceland, etc.) but discard 'tundra runoff' and 'river runoff'?
  + This does not have a total mass balance term, so I think we have to define anomaly from elsewhere (say the Mankoff total mass balance product, which is only Greenland main ice sheet). Then find the mean Bamber runoff during that baseline period, then subtract that mean from the rest of the Bamber time series to find the anomaly?

It may (or may not) help to view this graphic I made showing mass flow - this is Greenland not Antarctica. The numbers here are approximate and some are just guesses/placeholders. Ideally each number should have a reference. It is unlikely the equation will balance, and there should be losses (and gains?) to/from 'uncertainty'. There are issues with this figure - it combines process with (e.g., rain, snow) with form (solid, liquid). Also "melt" is conceptually downstream of SMB but should be physically downstream. 

![image](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/145117/477cb044-7a51-44e1-bdc8-497a0d7e1437)

**Comment by jlbamber** [comment]
Hi Ken, on a train so may not be super coherent. I won't address part I about anomaly vs. mass balance. But from memory of the discn we had during the workshop we talked about a reference period to use (say 1961-1990 for GrIS) which is close to steady state (not necessarily mass balance). For the GrIS there is a whole debate about what is meant by SS as there is a secular trend due to LIA age response from around 1900 onward but this is relatively small (order of magnitude smaller) than present day imbalance and I would say within uncertainties so I think we can define a refe period for anomalies. 

However, I am not sure we MUST doe these things now. For our new data set we have separate fields for tundra, GIC, PGIC, discharge and runoff so the user can choose exactly what they want to use. If they want GrIS only no problem. If they want Arctic land ice also no problem. We even partition runoff above/below the ELA aurguing that above the ELA it is ~ seasonal snowfall and below ice melt. So we propose to keep the data set comprehensive and flexible to use. 

In addition, anyone can calculate a FWF anomaly from the data set and choose any reference period they like. The anomaly would not be a mass balance one but a FWF one which is surely what we are aiming for. If the AIS folk take a similar approach then I don't see that there should be a big problem but we should definitely make sure it is crystal clear what the fields are and how to use them.
